compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/canonicalize_doubly_strided_op_hardware_aware.mlir

```diff
@@ -0,0 +1,53 @@
+// RUN: iree-opt --pass-pipeline="builtin.module(func.func(iree-amdaie-canonicalize-doubly-strided-op{hardware-aware=true},canonicalize))" --split-input-file -allow-unregistered-dialect --verify-diagnostics %s | FileCheck %s
```

Comment:
Do you need to have `canonicalize` in the test pass pipeline?

Reply:
Not needed, so I removed it.